WORKSPACE

```diff
@@ -144,10 +144,16 @@ load("@rules_jvm_external//:defs.bzl", "maven_install")
 # Note to developers: new dependencies should be added to //third_party:versions.bzl, not here.
 maven_install(
     artifacts = DAGGER_ARTIFACTS + get_maven_dependencies(),
+    fail_if_repin_required = True,
     fetch_sources = True,
+    maven_install_json = "@rules_jvm_external//third_party:maven_install.json",
```

Comment:
Shouldn't this be ``//third_party:maven_install.json`` since the file is part of our repository?

Reply:
I am still getting the same error.
![Screenshot from 2021-06-28 21-47-39](https://user-images.githubusercontent.com/54636525/123670145-867e1c00-d85a-11eb-98a2-3d691eef546b.png)

Reply:
Done. Thanks for pointing this out.